### ISSUE TITLE
Review JOSS

### ISSUE THREAD

**Issue by scemama** [NLESC-JCER/QMCTorch#145]
Hi! 
Here are a few comments to help improve the paper:

- [ ] line32 : varioationally -> variationally
- [ ] line43: the $D^{\uparrow}$ and $D^{\downarrow}$ determinants need to be indexed by $n$. You can do:
- [ ] $\sum_n c_n D_n^{\uparrow} D_n^{\downarrow}$, but this would imply a non-unique set of up- or down-determinants. Otherwise you can use  $\sum_{ij} c_{ij} D_i^{\uparrow} D_j^{\downarrow}$, but I am not sure that this is the representation you are using in the code.
- [ ] lines44->64 : for vector quantities, I would use bold face. For example, I would do: `$\mathbf{q}_i = \mathbf{r}_i + \sum_{i \ne j} K_{BF} (r_{ij})\,(\mathbf{r}_i - \mathbf{r}_j)$`.
- [ ] line54: You are using $\omega$ in the backflow expression, but you use also $\omega$ in the expression of the Jastrow line 49. You should use different letter to avoid confusion.
- [ ] line67: I would use a dot instead of `\times` for matrix multiplication of AO with $W_{SCF}^T$
- [ ] line86: $\rho$ is commonly used for the one-electron density. For the $N$-electron density, some authors use $\Pi$.
- [ ] line163: In the citation of Wheeler, there is some html `<tt>`

**Comment by scemama** [comment]
- [ ] In the documentation, to test the software `cd test` should be replaced by `cd tests`

**Comment by scemama** [comment]
- [ ] You might want also to remove the disclaimer from the README to match with the requirements of JOSS ;-)

**Comment by AbdAmmar** [comment]
Hello ! In addition to Anthony's comments, 

It would be helpful to include further references in Line 25

- [ ]  Choo et al 2020, DOI: 10.1038/s41467-020-15724-9
- [ ] Yang et al 2020, DOI: 10.1021/acs.jctc.9b01132
- [ ] Han et al 2019, DOI: 10.1016/j.jcp.2019.108929
- [ ] Kessler et al 2021,  DOI: 10.1002/adts.202000269
- [ ] Inui et al 2021, DOI: 10.1103/PhysRevResearch.3.043126
- [ ] Schätzle et al 2021, DOI: 10.1063/5.0032836
- [ ] Lin et al 2023, DOI: 10.1016/j.jcp.2022.111765

and in the "Backflow Transformation" section, Line 52 
- [ ] Schmidt et al 1981, "Title of the Paper," DOI: 10.1103/PhysRevLett.47.807

- [ ] Providing some insight into the computational cost difference between using automatic differentiation and analytical expressions for the calculation of the Laplacian of the Slater determinants would be interesting. This could help readers understand the trade-offs involved in the chosen approach.

**Comment by scemama** [comment]
> @scemama Thanks a lot for all the corrections, I've implemented all of them in #148 For the determinants I've used   
> ∑
> �
> �
> �
> �
> �
> ↑
> �
> �
> ↓
>  as it is what I have implemented in the code. I could implement   
> ∑
> �
> �
> �
> �
> �
> �
> �
> ↑
> �
> �
> ↓
> . Would you recommend that ?

Yes, if you have large CI expansions! You would only need to compute the unique set of $D_i^\uparrow$ and $D_j^\downarrow$ and thir gradients and Laplacian, and then you can recombine everything to get the wave function. This his how we obtain a $\sqrt{N}$ scaling for near-FCI wave functions. You can have a look at this presentation: http://irpf90.ups-tlse.fr/files/pacifichem.pdf or this paper: https://arxiv.org/abs/1510.00730

**Comment by tonnylou44853** [comment]
Hi!

In addition to the comments above, here is a list of typos I found in the docs:
- [ ] "Jastro Kernel" should be "Jastrow Kernel" in the second line of the first paragraph in https://qmctorch.readthedocs.io/en/latest/rst/qmctorch.html.
- [ ] I believe this should be a LaTeX symbol instead of plain text: `:math: K_{ee}` in the first paragraph of https://qmctorch.readthedocs.io/en/latest/rst/qmctorch.html#two-body-jastrow-factors?
- [ ] First line in https://qmctorch.readthedocs.io/en/latest/notebooks/wfopt.html#Running-the-wave-function-optimization should be **50** steps instead of 250 steps, as it does not matches with the code below.
- [ ] https://qmctorch.readthedocs.io/en/latest/notebooks/geoopt.html#Geometry-Optimization: First paragraph, second line "firs" -> "first".
- [ ] https://qmctorch.readthedocs.io/en/latest/notebooks/geoopt.html: Cell in last line is empty.

**Comment by NicoRenaud** [comment]
> > @scemama Thanks a lot for all the corrections, I've implemented all of them in #148 For the determinants I've used
> > as it is what I have implemented in the code. I could implement
> > . Would you recommend that ?
> 
> Yes, if you have large CI expansions! You would only need to compute the unique set of Di↑ and Dj↓ and their gradients and Laplacians. Then you can recombine everything to get the total wave function. This his how we obtain in QMC=Chem a O(Ndet) scaling for near-FCI wave functions. You can have a look at this presentation: http://irpf90.ups-tlse.fr/files/pacifichem.pdf or this paper: https://arxiv.org/abs/1510.00730 we can also have a zoom to if you want more detail :-)

Thanks for the explanation @scemama  ! That sounds like something I will have to do indeed :) I won't have too much time to develop new features on qmctorch before the end of the year. But I will pick this up in January ! I will come back to you ! Thanks !

**Comment by NicoRenaud** [comment]
> Hi!
> 
> In addition to the comments above, here is a list of typos I found in the docs:
> 
> * [x]  "Jastro Kernel" should be "Jastrow Kernel" in the second line of the first paragraph in https://qmctorch.readthedocs.io/en/latest/rst/qmctorch.html.
> * [x]  I believe this should be a LaTeX symbol instead of plain text: `:math: K_{ee}` in the first paragraph of https://qmctorch.readthedocs.io/en/latest/rst/qmctorch.html#two-body-jastrow-factors?
> * [x]  First line in https://qmctorch.readthedocs.io/en/latest/notebooks/wfopt.html#Running-the-wave-function-optimization should be **50** steps instead of 250 steps, as it does not matches with the code below.
> * [x]  https://qmctorch.readthedocs.io/en/latest/notebooks/geoopt.html#Geometry-Optimization: First paragraph, second line "firs" -> "first".
> * [x]  https://qmctorch.readthedocs.io/en/latest/notebooks/geoopt.html: Cell in last line is empty.

Thanks @tonnylou44853  ! I've made all the change and will merge as soon as the test are ok ! :) Thanks for all the input

**Comment by NicoRenaud** [comment]
Thanks a lot everyone for all the input !! I'm closing that issue now :)
